web/components/admin/Toolbar/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Toolbar extends React.Component {
+  render() {
+    const { applicantType, switchApplicantType, className } = this.props;
+    return (
+      <div className={`toolbar pad-sides-s pad-top-s fill-width ${className}`}>
+        <i>TODO: Toolbar component</i> <br />
+        Collection: {applicantType}
+        <button type="button" onClick={() => switchApplicantType('hacker')}>Hacker</button>
```

Comment:
for future reference, I think it would be nice to use something like this for switching applicant type: https://evergreen.segment.com/components/segmented-control

Reply:
I used the buttons for testing switching between applicants haha, it's supposed to be a select field according to the mocks. I've changed it to use our `<Select>` component and styled it to match the mocks.

Also I'm totally for using third-party components if they're well-maintained and don't look out of place in our current UI (or they do look out of place, but make up for it by providing extra functionality that we currently don't have / can't implement in time). I get the feeling that we'll end up using some for the filter panel.